### PR TITLE
Hurlburb/newtonsoft 13 0 1

### PR DESCRIPTION
**Pull request by brandonh-msft**
Newtonsoft.JSON needs updating due to a security vulnerability.

Additionally, try-convert now builds for net6.0, so docs needed a minor update

### CODE REVIEW

File: src/MSBuild.Abstractions/MSBuild.Abstractions.csproj [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <PackageReference Include="Microsoft.Build.Locator" Version="$(MicrosoftBuildLocatorVersion)" />
     <PackageReference Include="Microsoft.Build" Version="$(MicrosoftBuildVersion)" ExcludeAssets="runtime" />
     <PackageReference Include="Microsoft.Win32.Registry" Version="$(MicrosoftWin32RegistryVersion)" />
-    <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonVersion)" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```
Comment (jmarolf) [inline src/MSBuild.Abstractions/MSBuild.Abstractions.csproj:11]: why is this necessary?

Comment (brandonh-msft) [reply src/MSBuild.Abstractions/MSBuild.Abstractions.csproj:11]: https://github.com/advisories/GHSA-5crp-9r3c-p9vr

Comment (brandonh-msft) [reply src/MSBuild.Abstractions/MSBuild.Abstractions.csproj:11]: n/m I think what you're saying is why is it necessary given the change to `Versions.props` 👍🏻 will remove.

**Review by jmarolf — APPROVED** [review]
